src/Appwrite/Event/Event.php

```diff
@@ -24,11 +24,22 @@ class Event
     public const FUNCTIONS_QUEUE_NAME = 'v1-functions';
     public const FUNCTIONS_CLASS_NAME = 'FunctionsV1';
 
+    /** remove */
```

Comment:
leftover?

Reply:
nope, this should be removed once old queue is empty, and everything is moved to new renamed queue.